server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -173,6 +185,10 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
 
     public handler(rawMessage: IQueuedMessage) {
         let kafkaCheckpointMessage: IQueuedMessage | undefined;
+        const lumberJackMetric = Lumberjack.newLumberMetric(LumberEventName.DeliHandler);
```

Comment:
Could a new `IDeliServerConfiguration` option be added to enable / disable this telemetry logging? I would want to investigate the performance before enabling this

Reply:
I added a config at the level of IServerConfiguration. Please do let me know if you think a more granular microservice level config will be useful. Also I set this to true in configuration.ts. Would you prefer that it be turned off by default?

Reply:
I think having it on by default is okay.

I would rather have the config setting prevent `const lumberJackMetric = Lumberjack.newLumberMetric(LumberEventName.DeliHandler);` from being called. If the setting is off, `lumberJackMetric` should be `undefined`.